src/__tests__/__snapshots__/ItemPresentation.test.jsx.snap

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ItemPresentation component matches snapshot 1`] = `
+<DocumentFragment>
+  <div
+    class="item-presentation"
+  >
+    <div
+      class="item-details"
+    >
+      <h2>
+        Overview
+      </h2>
+      <p>
+        <strong>
+          Name:
+        </strong>
+         Sample Item
+      </p>
+      <p>
+        <strong>
+          Type:
+        </strong>
+         Sample Quality
+      </p>
+      <p>
+        <strong>
```

Comment:
- [ ] [OPTIONAL] Great job on your app your UI is great and the app is cool however there are some cards that upon clicking leads to an error and the error is caused by empty array iteration. I would suggest handling  this error by putting a checks before iterating any array to see if it contains any item before iterating it. See screenshot of error:

<img width="1502" alt="Screenshot 2023-10-14 at 6 31 50 PM" src="https://github.com/lily4178993/dota2-insight-hub/assets/44798044/bcf3f280-3683-45f7-a0bb-7c4f2e6473e7">

Reply:
I will implement this in the near future.